src/taipy/config/checker/_checkers/_gLobal_config_checker.py

```diff
@@ -18,17 +18,30 @@
 
 
 class _GlobalConfigChecker(_ConfigChecker):
+    
+    _ACCEPTED_REPOSITORY_TYPES = ["filesystem", "sql"]
```

Comment:
```suggestion
    _ACCEPTED_REPOSITORY_TYPES = {"filesystem", "sql"} # or set()
```

@jrobinAV @bobbyshermi
I'm thinking that why don't we keep this list empty? Then on taipy-core, we will propagate the list by injecting available repo options. This way, it's easier to expand the supported repo type (different packages provide different supported repo types) while still helps keep the maintenance of possible repo types within their respective package. Also I think a set would be better as we would want the value to be unique.